napari/utils/_tracebacks.py

```diff
@@ -27,6 +29,10 @@ def get_tb_formatter() -> Callable[[ExcInfo, bool], str]:
         def format_exc_info(
             info: ExcInfo, as_html: bool, color='Neutral'
         ) -> str:
+            # avoids printing the array data
+            np.set_string_function(
```

Comment:
two thoughts:
a) is there a way to get the current string_function?  (so that we can restore it in case a user has already set it?)

b) we probably want to be extra careful not to set this on someone and have it be permanent... so probably best to put everything between this line and the `set_string_function(None)` line in a `try/finally` clause to make sure that we set it back to whatever it was before

Reply:
I could not find a way to get it back, I think it is not exposed to the NumPy Python API :(

Reply:
yeah, bummer... some discussion here: https://github.com/numpy/numpy/issues/11266